signer/src/bitcoin/rpc.rs

```diff
@@ -197,7 +197,7 @@ pub struct BitcoinTxVin {
     /// This field is omitted if block undo data is not available, so it is
     /// missing whenever the `fee` field is missing in the
     /// [`BitcoinTxInfo`].
-    pub prevout: BitcoinTxVinPrevout,
+    pub prevout: Option<BitcoinTxVinPrevout>,
```

Comment:
nit: should we add to the comment that this is none for coinbase?

Reply:
Good point.

Reply:
Ohhh wait, maybe we do not want to merge this actually 🤔.